article/_posts/2019-10-17-StraightToThePoint.md

```diff
@@ -0,0 +1,80 @@
+---
+layout: review
+title: "Straight to the point: reinforcement learning for user guidance in ultrasound"
+tags: deep-learning reinforcement cardiac
+author: "Nathan Painchaud"
+cite:
+    authors: "Fausto Milletari, Vighnesh Birodkar, Michal Sofka"
+    title:   "Straight to the point: reinforcement learning for user guidance in ultrasound"
+    venue:   "MICCAI 2019 (SUSI)"
+pdf: "https://arxiv.org/pdf/1903.00586.pdf"
+---
+
+
+# Highlights
+- System to guide inexperienced users towards the acquisition of clinically relevant images of the heart in ultrasound
+- Environments to simulate offline any (reasonable) acquisition trajectory the user may take while scanning the patient
+  by making use of tracked video sequences
+- Comparison of the performances of the method with the ones obtained by training a classifier to learn a policy on the
+  same data in a fully supervised manner
+
+
+# Introduction
+The authors propose a method for guiding inexperienced users in the acquisition through the parasternal long axis (PLAx)
+sonic window on the heart. The degrees of freedom covered by the guidance are detailed in Figure 2.
+
+![](/article/images/StraightToThePoint/figure2.png)
+
+
+# Methods
+
+## Environment
+It would be impossible to collect data according to the RL agent's policy since it would require to scan a patient
```

Comment:
What's the link between wanting to guide users and training an RL agent ? Do they want the users out of the process and automate it instead ? Is the agent going to guide the users ? I think it needs a little more introduction

Reply:
I detailed the introduction (rather than the "Environment" section) to explain better the relevance of guiding users during the acquisition, and what exactly it is that the RL policy learns to predict.